Example/AccessibilitySnapshot/SwiftUIView.swift

```diff
@@ -77,8 +77,27 @@ struct SwiftUIView: View {
                 .accessibility(label: Text("Label"))
                 .accessibility(value: Text("Value"))
                 .accessibility(hint: Text("Hint"))
-
-            Spacer()
+            
+            if #available(iOS 14.0, *) {
+                // View with label, value, hint, and Custom Actions.
+                Circle()
+                    .accessibility(label: Text("Label"))
+                    .accessibility(value: Text("Value"))
+                    .accessibility(hint: Text("Hint"))
+                    .accessibilityAction(named: "Custom") {}
+            }
+// Swift 5.5 ships with Xcode 13 and the iOS 15 SDK.
+#if swift(>=5.5)
```

Comment:
#197 should fix this so you don't need the Swift version checks for 5.4 or 5.5

Reply:
Now only needing a single check in the accessibilityParser. Much nicer. Thanks